tests/test_func.py

```diff
@@ -26,3 +48,41 @@ def test_get_chain_meta():
     assert chain.name == "Arbitrum One"
     assert chain.nativeCurrency["symbol"] == "ETH"
     assert len(chain.explorers) > 0
+
+
+def test_get_rpcs_alchemy(env):
+    alchemy_key = "alchemistsayswhat"
+    expected_rpc = f"https://arb-mainnet.g.alchemy.com/v2/{alchemy_key}"
```

Comment:
looking at the regex, i thought it needed the `$` like `${alchemy_key}`

Reply:
think `expected_rpc` is what the final should look like

Reply:
oh duhh, the f-str and `{` was confusing me lol